sql-plugin/src/main/scala/com/nvidia/spark/rapids/DateTypeUtils.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids
+
+import org.apache.spark.sql.types._
+
+/**
+ * Class for helper functions for Date and Timestamp
+ */
+object DateTypeUtils {
```

Comment:
Since we  are checking for both date and time, let's call it `DateTimeUtils`.

Reply:
Wait, since we already have `DateUtils.scala`, I would recommend putting this `DateTimeUtils` class code into it, and renaming `DateUtils.scala` into `dateTimeUtils.scala` too.

Reply:
Done.